tests/wavelet_tests.py

```diff
@@ -22,7 +22,7 @@ def test_morse_wavelet_transform_real(self):
         length = 1023
         radian_frequency = 2 * np.pi / np.logspace(np.log10(10), np.log10(100), 50)
         x = np.random.random(length)
-        wtx = morse_wavelet_transform(x, 3, 10, radian_frequency)
+        wtx = morse_wavelet_transform(x, 3, 10, radian_frequency, False)
```

Comment:
Why can't we have the complex=False a default for this function. You seem to have it changed to a complusory argument?

Reply:
I will revert to have this default put back

Reply:
I think this was forgotten.